mne_bids/write.py

```diff
@@ -783,6 +784,10 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
                  while warning the user. For EEG and iEEG data, conversion will
                  be to BrainVision format, for MEG conversion will be to FIF.
 
+    .. warning:: In order to ensure anonymization, files are converted to FIF.
+                 In practice, this means if you pass a brainvision EEG data
+                 file, it will be converted to FIF and back to brainvision.
```

Comment:
but why is this necessary? we can write to Brainvision

Reply:
But we have to anonymize in FIF or so I thought

Reply:
okay I am lost :)

Reply:
Haha okay so to anonymize we have to be in FIF but to save for EEG or iEEG we can't be in FIF. Therefore we have to convert and convert back unless you want to start dealing with mne.io.Raw objects and not going off disc (see comment above). I can talk for the next 15 minutes if you want to call and that helps.

Reply:
but what do you mean by "we have to be in FIF" ?